catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/config/JdbcConfig.java

```diff
@@ -13,59 +13,60 @@
 import org.apache.commons.lang3.StringUtils;
 
 public class JdbcConfig extends Config {
+  private static final String VERSION_0_3_0 = "0.3.0";
```

Comment:
Can we put the version string into a common place like `Config`? So that we don't have to define it in several places.

Reply:
Sure, you reminded me, thx